Add docs for OpenTelemetry Java (#5968)
getsentry/sentry-docs: src/platform-includes/performance/opentelemetry-setup/with-java-agent/with-auto-init/debugging/java.mdx

> To enable debug logging for Sentry, please provide `SENTRY_DEBUG=true` as environment variable or
> add `debug=true` to your `sentry.properties`.
> 
> To also show debug output for OpenTelemetry please add `-Dotel.javaagent.debug=true` to the `java` command.

Reviewer: ```suggestion
To enable debug logging in Sentry, set `SENTRY_DEBUG=true` as an environment variable or
add `debug=true` to your `sentry.properties`.

To show debug output for OpenTelemetry, add `-Dotel.javaagent.debug=true` to the `java` command.
```
